packages/aa-infra/src/custom/alchemyGasManagerHooks.ts

```diff
@@ -137,139 +77,220 @@ export type AlchemyGasManagerHooks = {
   };
 };
 
+export type AlchemyGasManagerConfig = {
+  client: Client<Transport, Chain, SmartAccount>;
```

Comment:
We don't need a hoisted account here, and this breaks type inference in the example
```suggestion
  client: Client
```